### PR TITLE
test: added helm cli integration tests

### LINKED ISSUE #601: Add unit- and integration tests for helm support
related to #590

### PR DESCRIPTION
**Pull request by adietish**
fixes #601 
depends on #606

### CODE REVIEW

File: src/test/java/org/jboss/tools/intellij/openshift/actions/helm/UninstallReleaseActionTest.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
```
Comment (sbouchet) [inline src/test/java/org/jboss/tools/intellij/openshift/actions/helm/UninstallReleaseActionTest.java:2]: should be 2023

**Review by sbouchet — COMMENTED** [review]
minor change on comment

**Comment by sbouchet** [comment]
please also fix the integration tests ( no helm repository found )

**Comment by adietish** [comment]
> please also fix the integration tests ( no helm repository found )

That's what I dont understand. This error exists on main but is fixed in this PR: https://github.com/redhat-developer/intellij-openshift-connector/pull/602/files#diff-a6c2a5e0c355b5d4a9b7a5c78a9aa8c4a6912cc957dc5b7a634623eb03116f48R29

It is as if integration tests are run against main and not against this PR/branch.

**Comment by adietish** [comment]
@sbouchet please re-review

**Comment by adietish** [comment]
@sbouchet please review.

**Comment by adietish** [comment]
the IT failures were caused by CRDs required for vertx missing in kind. Switched to kuberos which worked for me on a local kind cluster.

**Review by sbouchet — APPROVED** [review]
lgtm

**Pull request merged**